Analysis/analysis.tex

```diff
@@ -57,17 +59,18 @@ \subsection{Pileup Reweighting}
   }
   \label{fig:pudist}
 \end{figure}
+\PH{This setion is completely nebulous, you don't actually expalin that you are deriving a weight. This needs a rewrite}
```

Comment:
I think the new text explains we are applying a weight to obtain a desired PU spectrum. I really don't want to go into too much detail here